yeelight_light_fancl5.yaml

```diff
@@ -136,7 +226,64 @@ xiaomi_ylyk01yl:
             args: ['int(keycode)']
 
   on_long_press:
-    # Trigger on any keycode
+    # night light
+    - keycode: 1
+      then:
+       - light.toggle: night_light
+       - switch.turn_on: buzzer_beep
+       
+    # stepless cycle control speed
+    - keycode: 2
+      then:
+        - fan.cycle_speed: yeelight_ceiling_fan
+        - switch.turn_on: buzzer_beep
+       
+    # fan mode REVERSE select for circle winds
+    - keycode: 4
```

Comment:
You are turning the fan on at long press (**keycode 4**)

Reply:
no, turn on fan using keycode 0

long press keycode 4 from change reverse mode, short press  keycode 4 for forward mode